### ISSUE TITLE
api key

### ISSUE THREAD

**Issue by i0rul** [x0rz/tweets_analyzer#61]
hi need help to get api key it's hard to create an api apps

**Comment by 0xh0n3y** [comment]
You will need to have a Twitter's Developer's account first yhen after you will get Twitter's APIs keys and then after you have to update your keys in secrets.py and thats all you need to do.

**Comment by i0rul** [comment]
yeah i tried to get twitter's developer's account but i couldn't submit one, maybe u help me

**Comment by i0rul** [comment]
> You will need to have a Twitter's Developer's account first yhen after you will get Twitter's APIs keys and then after you have to update your keys in secrets.py and thats all you need to do.

yeah i tried to get twitter's developer's account but i couldn't submit one, maybe u help me

**Comment by 0xh0n3y** [comment]
Once you got your Twitter's developer account ,you need to go to apps option on drop down menu opens when you click on your name and then hit enter on it . Then after directed to apps section where you get create app option then fill the form for your app and then after you will get your API's keys.

**Comment by 0xh0n3y** [comment]
You had to specify Twitter deveelopers on email that how will you be using API's keys .Provide them links to Repos on github which you be using with your API's keys and please be specific to them , then only you be getting your Twitter's developer Account.

**Comment by 0xh0n3y** [comment]
On providing Twitter info. about your usage of Twitter's API's you will definitely get access to your Twitter Developer Account.

**Comment by i0rul** [comment]
> On providing Twitter info. about your usage of Twitter's API's you will definitely get access to your Twitter Developer Account.

first of all thank you for replying 
i write to them this 
1. I will be using the twitter api to gather data for a group project in class.
2. I plan to analyze the content of the tweets to categorize them and understand regional trends and how to best visualize these trends.
3. I will not be tweeting, retweeting or liking any content with this application and will not be interacting with any twitter users.
4. Summarized and or aggregated tweets will be displayed as part of my project presentation.

but they send me this "Your Twitter developer account application was not approved."
i tried many time even i use my uni email which end by .edu

**Comment by 0xh0n3y** [comment]
You can use any email address :) .
Provide them screenshots of apps you will using with API keys or links like (e.g www.github.com/x0rz/tweets_analyzer ) you should mention in your reply to Twitter Developers and you have to be specific in your words for getting to access to Twitter developer account.

**Comment by 0xh0n3y** [comment]
Here a copy of email i send in reply toTwitter developer account [developer-accounts@twitter.com] :
Here is a copy:
------------------------------
Hi there, i will be doing only analysis of tweets with the Twitter's APIs on my apps developed using Twitter's APIs 

for which i will be using this software like https://github.com/x0rz/tweets_analyzer for which i need Twitter's APIs .

I will be only displaying the same analysis of Twitter tweets on my blogs like [mention your website where you want to blog].

And i will not be developing any app using Twitter's APIs to retweet, liking and tweeting .

I will only be tweeting that analysis on Twitter trends used by users of Twitter.

Here the software analysis can be seen on this link :https://twitter.com/x0rz/status/826007772122771456.

And moreover i will be developing software like this to display call graph of Twitter's trends used by Twitter's user https://twitter.com/fs0c131y/status/1105405045414219776. 

Please provide me Twitter's APIs as early as possible for developing my apps .

Thank you , Twitter Developer's.
------------------------------------------------------------- 
And after that i got access to my Twitter developer account and hence i get access to API keys.
You

**Comment by 0xh0n3y** [comment]
Reply with this email when first time you got email from developer-accounts@twitter.com and they contact through email.
